app/boards/shields/lily58/boards/nice_nano.overlay

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+&spi1 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	mosi-pin = <6>;
+	// Unused pins, needed for SPI definition, but not used by the ws2812 driver itself.
+	sck-pin = <5>;
+	miso-pin = <7>;
+
+	led_strip: ws2812@0 {
+		compatible = "worldsemi,ws2812-spi";
+		label = "WS2812";
+
+		/* SPI */
+		reg = <0>; /* ignored, but necessary for SPI bindings */
+		spi-max-frequency = <4000000>;
+
+		/* WS2812 */
+		chain-length = <1>; /* number of LEDs */
```

Comment:
Maybe set this to a reasonable/common value? I think I've seen kits sold with 5 for each side?

Reply:
The chain length is variable from what I've seen. I can make this 5 as a default just to be 'safe' as that seems like it may be a more common situation.